### PR TITLE
Added new hardware models and support for simulation / evolving

### PR DESCRIPTION
**Pull request by oliverweissl**
-> Modeling new hardware in progress: detailed measurements will be added still (weight, size)
-> no documentation yet for how to use new hardware
-> controller -> will be added once fixed cable comes

### CODE REVIEW

**Comment by surgura** [comment]
Good start. I think it would be best to completely separate into two modules: ModularRobot and ModularRobotV2 or some names like that. They can be similar or reuse stuff under the hood, but then it would be clear for the user which hardware they are targeting.

**Comment by oliverweissl** [comment]
Seperated ModularRobot into ModularRobotV2 for easier access to new hardware.

File: ci_group/revolve2/ci_group/modular_robots.py
Comment (surgura) [inline ci_group/revolve2/ci_group/modular_robots.py:1]: seems good and clearer to me. can we also seperate into a modular_robots_v2.py?

File: modular_robot/revolve2/modular_robot/_modular_robotv2.py
Comment (surgura) [inline modular_robot/revolve2/modular_robot/_modular_robotv2.py:1]: I personally would prefer a revolve2/modular_robot/v1/...
and revolve2/modular_robot/v2/...
and revolve2/modular_robot/_common for classes that are shared, but we only expose things explicitly in v1 and v2, so _common is private. user would then do revolve2.modular_robot.v2.Body/Brick/Core etc

**Pull request closed without merging**